examples/zephyr/README.md

```diff
@@ -1,34 +1,118 @@
-# Golioth Zephyr Examples
+# Golioth Firmware SDK: Zephyr Support (beta)
 
-This directory contains a set of examples intended to build and run with the latest release of
-Zephyr.
+Zephyr support in the Golioth Firmware SDK is in active development and
+currently designated as "beta". We will update this page when this port
```

Comment:
Adding more information on what "beta" means
```suggestion
currently designated as "beta". This SDK supports all the features of the Golioth platform, but is not yet as optimized as the Golioth Zephyr SDK. We will update this page when this port
```

Reply:
I like this